src/pyodide/internal/_workers.py

```diff
@@ -851,6 +874,27 @@ def __getattr__(self, name):
         setattr(self, name, result)
         return result
 
+    def _fetch_wrapper(self, fetcher, *args, **kwargs):
+        kwargs["fetcher"] = fetcher
+        return fetch(*args, **kwargs)
```

Comment:
What about:
```suggestion
    def fetch(self, *args, **kwargs):
        return fetch(*args, fetcher=self._binding.fetch, **kwargs)
```

Reply:
well, I think the name needs to remain `_fetch_wrapper` to distinguish it. No need to make it public either.